### PR TITLE
Feat/basic helm chart support

### PR DESCRIPTION
**Pull request by w3dani**
## What’s the focus of this PR
- Add another schematics for creating a basic helm chart templates
- Important: Not tested against a kubernetes cluster so it could still not be 100% stable, but it's a starting point to keep iteraring
- Updated documentation

## How to review this PR
- Run `schematics @guidesmiths/cuckoojs-schematics:helm`

Before submitting this PR, I made sure:
- [x] The code builds clean without any errors or warnings
- [x] I'm using our guidelines
- [x] I've added unit tests if feasible

**IMPORTANT: This PR has been created from last commit on `feat/ci-build-push-github-action` so please, let's complete first the other branch before reviewing or merging this one**

### CODE REVIEW

File: packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  "properties": {
+    "serviceName": {
+      "type": "string",
+      "description": "service name to deploy on kubernetes",
+      "x-prompt": "What is the name of the service you want to config deployment?"
+    },
+    "imageName": {
+      "type": "string",
+      "description": "docker image name to deploy on kubernetes",
+      "x-prompt": "What image you want to config deployment?"
```
Comment (inigomarquinez) [inline packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json:15]:
```suggestion
      "x-prompt": "What image do you want to config deployment?"
```

File: packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    },
+    "resourcesLimitCpu": {
+      "type": "number",
+      "description": "max cpu used by a pod",
+      "x-prompt": "What is the CPU limit per pod (specified en 'm' units)?",
```
Comment (inigomarquinez) [inline packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json:20]:
```suggestion
      "x-prompt": "What is the CPU limit per pod (specified in 'm' units)?",
```

File: packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      "default": 100
+    },
+    "resourcesLimitMemory": {
+      "type": "number",
+      "description": "max memory used by a pod",
+      "x-prompt": "What is the memory limit per pod (specified en 'Mi' units)?",
```
Comment (inigomarquinez) [inline packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json:26]:
```suggestion
      "x-prompt": "What is the memory limit per pod (specified in 'Mi' units)?",
```

File: packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      "default": 128
+    },
+    "resourcesRequestCpu": {
+      "type": "number",
+      "description": "requested cpu by a pod to be launched",
+      "x-prompt": "What is the CPU requested per pod (specified en 'm' units)?",
```
Comment (inigomarquinez) [inline packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json:32]:
```suggestion
      "x-prompt": "What is the CPU requested per pod (specified in 'm' units)?",
```

File: packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      "default": 100
+    },
+    "resourcesRequestMemory": {
+      "type": "number",
+      "description": "requested memory by a pod to be launched",
+      "x-prompt": "What is the memory requested per pod (specified en 'Mi' units)?",
```
Comment (inigomarquinez) [inline packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json:38]:
```suggestion
      "x-prompt": "What is the memory requested per pod (specified in 'Mi' units)?",
```

File: packages/@guidesmiths/cuckoojs-schematics/src/ci/schema.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  "properties": {
+    "ciProvider": {
+      "type": "string",
+      "description": "CI provider where ci will be set up",
+      "enum": [
+        "github",
+        "azuredevops"
+      ],
+      "x-prompt": {
+        "message": "What CI provider you want to set up?",
```
Comment (inigomarquinez) [inline packages/@guidesmiths/cuckoojs-schematics/src/ci/schema.json:15]:
```suggestion
        "message": "What CI provider do you want to set up?",
```

**Review by inigomarquinez — CHANGES_REQUESTED** [review]
Some minor changes suggested :)

**Comment by w3dani** [comment]
> Don't we want to offer the option of using KEDA instead HPA too?

using KEDA means we have to deploy previosly it on the cluster. For me is ok, but I'd go first for the most basic use case that is provided by kubernetes natively

**Review by inigomarquinez — APPROVED** [review]
LGTM!

**Review by neodmy — APPROVED** [review]

**Pull request merged**